src/coreclr/debug/daccess/dacdbiimpl.cpp

```diff
@@ -5199,7 +5199,7 @@ void DacDbiInterfaceImpl::Hijack(
     HRESULT hr = m_pTarget->GetThreadContext(
         dwThreadId,
         CONTEXT_FULL,
-        sizeof(ctx),
+        sizeof(DT_CONTEXT),
```

Comment:
Is DacDbiInterfaceImpl::Hijack is Windows-only and if 'yes', is this change needed?